src/qml/TrackerSettings.qml

```diff
@@ -0,0 +1,546 @@
+          font: Theme.defaultFont
+          wrapMode: Text.WordWrap
+          enabled: timeInterval.checked
+          visible: timeInterval.checked
+          Layout.leftMargin: 8
+          Layout.fillWidth: true
+        }
+
+        QfTextField {
+          id: timeIntervalValue
+          width: timeInterval.width
+          font: Theme.defaultFont
+          enabled: timeInterval.checked
+          visible: timeInterval.checked
+          horizontalAlignment: TextInput.AlignHCenter
+          Layout.preferredWidth: 60
+          Layout.preferredHeight: font.height + 20
+
+          inputMethodHints: Qt.ImhFormattedNumbersOnly
+          validator: DoubleValidator { locale: 'C' }
+
+          onTextChanged: {
+            if( text.length === 0 || isNaN(text) ) {
+              positioningSettings.trackerTimeInterval = NaN
+            } else {
+              positioningSettings.trackerTimeInterval = parseFloat( text )
+            }
+          }
+        }
+
+        Label {
+          text: qsTr("Activate distance constraint")
+          font: Theme.defaultFont
+          wrapMode: Text.WordWrap
+          Layout.fillWidth: true
+
+          MouseArea {
+            anchors.fill: parent
+            onClicked: minimumDistance.toggle()
+          }
+        }
+
+        QfSwitch {
+          id: minimumDistance
+          Layout.preferredWidth: implicitContentWidth
+          Layout.alignment: Qt.AlignTop
+          checked: false
+          onCheckedChanged: {
+            positioningSettings.trackerMinimumDistanceConstraint = checked
+          }
+        }
+
+        DistanceArea {
+          id: infoDistanceArea
+          project: qgisProject
+          crs: qgisProject ? qgisProject.crs : CoordinateReferenceSystemUtils.invalidCrs()
+        }
+
+        Label {
+          text: qsTr("Minimum distance [%1]").arg( UnitTypes.toAbbreviatedString( infoDistanceArea.lengthUnits ) )
+          font: Theme.defaultFont
+          wrapMode: Text.WordWrap
+          enabled: minimumDistance.checked
+          visible: minimumDistance.checked
+          Layout.leftMargin: 8
+          Layout.fillWidth: true
+        }
+
+        QfTextField {
+          id: minimumDistanceValue
+          width: minimumDistance.width
+          font: Theme.defaultFont
+          enabled: minimumDistance.checked
+          visible: minimumDistance.checked
+          horizontalAlignment: TextInput.AlignHCenter
+          Layout.preferredWidth: 60
+          Layout.preferredHeight: font.height + 20
+
+          inputMethodHints: Qt.ImhFormattedNumbersOnly
+          validator: DoubleValidator { locale: 'C' }
+
+          onTextChanged: {
+            if( text.length === 0 || isNaN(text) ) {
+              positioningSettings.trackerMinimumDistance = NaN
+            } else {
+              positioningSettings.trackerMinimumDistance = parseFloat( text )
+            }
+          }
+        }
+
+        Label {
+          text: qsTr("Activate sensor constraint")
```

Comment:
What exactly are "sensor constraints"? Down there we discuss just "constraints" are they the same?

Reply:
@suricactus , I've updated to "activate new sensor data constraint".